draft-gbb-ccamp-optical-path-computation-yang.md

```diff
@@ -211,11 +211,20 @@ flexi-grid-label-start-end, flexi-grid-label-hop and flexi-grid-label-step defin
 
 # Manageability Considerations
 
-  TBD. 
+This document provides a method for requesting path computations for WSON and Flexi-Grid tunnels. Consideration of mechanisms to gather and collate information required for the path computations will be necessary. Furthermore, storing path computation requests and responses and triggering actions according to configured will also need to be carefully managed and secured.
+
```

Comment:
....triggering actions according to configured will also need....
I would think the words missing would be "tunnels" probably but the sentence has to be completed